chunky/src/java/se/llbit/chunky/ui/render/tabs/WaterTab.java

```diff
@@ -132,21 +132,23 @@ public Node getTabContent() {
   @Override
   public void initialize(URL location, ResourceBundle resources) {
     waterVisibility.setName("Water visibility");
-    waterVisibility.setTooltip("Visibility depth under water.");
+    waterVisibility.setTooltip("Distance of visibility past the surface of the water.");
     waterVisibility.setRange(0, 50);
     waterVisibility.clampMin();
     waterVisibility.onValueChange(value -> scene.setWaterVisibility(value));
 
     waterOpacity.setName("Water opacity");
-    waterOpacity.setTooltip("Sets how opaque the water surface appears.");
+    waterOpacity.setTooltip("Opacity of the surface of the water.");
```

Comment:
```suggestion
    waterOpacity.setTooltip("Opacity of the water surface.");
```

Reply:
What is wrong with this tooltip? The tooltip for `Water visibility` also uses "surface of the water".

Reply:
Just personal taste, I find "water surface" better than "surface of the water".

Reply:
I find that the opposite is better, but I'll change it. "Water surface" is just as understandable and takes up less space.